CMakeModules/FindCLANG_FORMAT.cmake

```diff
@@ -20,7 +20,8 @@
 #
 
 find_program(CLANG_FORMAT_EXECUTABLE
-             NAMES clang-format
+             NAMES clang-format-11
+                   clang-format
```

Comment:
```suggestion
             NAMES clang-format
                   clang-format-11
```

Reply:
no this is not good, because on many distros clang-format is an alias for a specific clang-format version, which is then found first. Finding the specific version number you want is better.

Reply:
On most distros `clang-format` points to the latest version, which is basically what we want. Looking for a specific version first might let cmake find a non-standard/outdated version first.

Reply:
on ubuntu it does not

Reply:
Ubuntu should runs `update-alternatives` when install a new version of `clang-format` (or any multi-version package) and updates it to latest version unless you picked a version yourself before, or does it not?

Reply:
On `ghcr.io/votca/buildenv/ubuntu:rolling`:
```
$ ls -l /usr/bin/clang-format
lrwxrwxrwx 1 root root 15 Sep 17 07:34 /usr/bin/clang-format -> clang-format-11
```